ddtrace/debugging/_debugger.py

```diff
@@ -246,6 +248,12 @@ def __init__(self, tracer=None):
         )
 
         # Register the debugger with the RCM client.
+        if asbool(os.environ.get("DD_REMOTE_CONFIGURATION_ENABLED", True)) is False:
+            os.environ["DD_REMOTE_CONFIGURATION_ENABLED"] = "true"
```

Comment:
Can we add a comment here saying that this is only temporary and will be reverted once the `DD_REMOTE_CONFIGURATION_ENABLED` variable has been removed, please?

Reply:
Same as above. Until we have confirmation of when it'll be removed, this PR should not add that comment IMO.